aws/aws.go

```diff
@@ -103,6 +103,8 @@ func ConfigFromURLParams(q url.Values) (*aws.Config, error) {
 			}
 			cfg.S3ForcePathStyle = aws.Bool(b)
 		case "awssdk":
+		case "ssetype":
```

Comment:
These shouldn't be needed here. The caller should handle and remove any known query parameters before calling this function.

Can you revert edits from this file, and then edit s3blob.go to do that? https://github.com/google/go-cloud/blob/6d5d2890816abc88cd11ef13d6cd6506eb5b5537/runtimevar/awsparamstore/awsparamstore.go#L142 is an example.